features/mbedtls/importer/adjust-config.sh

```diff
@@ -148,6 +148,8 @@ conf set MBEDTLS_MPI_MAX_SIZE     512
 
 # The following configurations are needed for Mbed Crypto.
 # They are related to the persistent key storage feature.
+conf set MBEDTLS_PSA_CRYPTO_C
```

Comment:
Ok, so if we back up a little bit, I think the motivation for this was [us observing](https://github.com/ARMmbed/mbed-os/pull/9779#issuecomment-466962692) that `check_config.h` didn't like that `MBEDTLS_PSA_CRYPTO_STORAGE_C` was enabled but some of its prerequisites (which include `MBEDTLS_PSA_CRYPTO_C`) were not.

So if I understand your comment correctly, it would probably be safer to go about it the other way round: keep `MBEDTLS_PSA_CRYPTO_C` disabled by default, but also disable by default everything that depends on it, to keep `check_config.h` happy. @Patater wdyt?

Reply:
Yes, please keep `MBEDTLS_PSA_CRYPTO_C` (and things that depend on it) disabled by default. Specific PSA targets enable it in `targets.json`, and specific applications that can guarantee an entropy source enable it in `mbed_app.json` or a custom Mbed TLS configuration.